src/renderer/components/outputs.tsx

```diff
@@ -0,0 +1,87 @@
+import { observer } from 'mobx-react';
+import * as MonacoType from 'monaco-editor';
+import * as React from 'react';
+
+import { AppState } from '../state';
+import { activateTheme } from '../themes';
+import { Output } from './output';
+
+const defaultMonacoOptions: MonacoType.editor.IEditorOptions = {
+  minimap: {
+    enabled: false,
+  },
+  wordWrap: 'on',
+};
+
+interface OutputsProps {
+  appState: AppState;
+}
+
+interface OutputsState {
+  monaco?: typeof MonacoType;
+  isMounted?: boolean;
+  monacoOptions: MonacoType.editor.IEditorOptions;
+}
+
+@observer
+export class Outputs extends React.Component<OutputsProps, OutputsState> {
+  constructor(props: OutputsProps) {
+    super(props);
+
+    this.state = { monacoOptions: defaultMonacoOptions };
+  }
+
+  /**
+   * Executed right after the component mounts. We'll setup the IPC listeners here.
```

Comment:
Are there IPC listeners here?

Reply:
oops, this snuck in before I deleted some code :P

Reply:
addressed in newest commit